src/interface.jl

```diff
@@ -1,12 +1,24 @@
-# TODO: Improve docstring https://docs.julialang.org/en/v1/manual/documentation/
-"stage_states(mdp::MDP, epoch::Int64)"
+"""
```

Comment:
I think it would be great if usage examples were shown in the interface documentation, e.g. 
```
     stage(s::MDPState)
Return number of state's stage
````

Reply:
The documentation contained examples few commits back, then you pointed out that the functions in the interface do not have input arguments specified, so I removed them.

Reply:
Oh ok, I think that might have been a misunderstanding. I took issue with some of the examples having the very specific input types, such as FHWrapper. My point was that the example should not be so specific if the function is supposed to be defined by the suer for some generic user-specified mdp. Sorry about the confusion.